vllm/v1/core/kv_cache_utils.py

```diff
@@ -280,7 +327,7 @@ def hash_request_tokens(block_size: int,
             "The number of multi-modal positions and hashes must match.")
 
     # TODO: Extend this to support other features such as LoRA.
-    need_extra_keys = bool(mm_positions)
+    need_extra_keys = bool(mm_positions) or (request.lora_request is not None)
```

Comment:
1. Please help remove the above TODO
2. Maybe it's more clean to make this as a separate function.

```python
def need_extra_keys(request: Request) -> bool:
    return request.mm_positions or request.lora_request is not None
```